INSTALL.md

```diff
@@ -1,120 +1,139 @@
+**NOTE:** The instructions below assume you're installing 64-bit MSYS2 and using the 32-bit MinGW toolchain (they work with other configurations, but the commands won't be exactly the same).
+
+- Choice of 32-bit vs 64-bit for MSYS2 doesn't really matter; either is fine
+- There's currently no benefit to building a 64-bit version of libgambatte on Windows, so 32-bit is used for the release binaries
 
-***\*NOTE:*** At the end of these steps, to do any build tasks related to Gambatte-Speedrun, open the MSYS2 MinGW 32-bit shell (under the MSYS2 folder in the Start menu; ***\*must be\**** this specific shell in order for building to work).
+**NOTE:** At the end of these steps, to do any build tasks related to libgambatte, open the MSYS2 MinGW 32-bit shell (under the MSYS2 folder in the Start menu; **\*\*must be\*\*** this specific shell in order for building to work).
 
 ### Basic steps
 
-*Do the following:*
+Do the following:
 
-\- Install [MSYS2](https://www.msys2.org/) by selecting the one-click installer exe for x86_64
+- Install [MSYS2](https://www.msys2.org/) by selecting the one-click installer exe for x86_64
 
-\- Run MSYS2 shell and update MSYS2 core components and packages *(copied from [here](https://www.msys2.org/wiki/MSYS2-installation/#iii-updating-packages))*:
+- Run MSYS2 shell and update MSYS2 core components and packages _(copied from [here](https://www.msys2.org/wiki/MSYS2-installation/#iii-updating-packages))_:
+
+```
+pacman -Syuu
 ```
-$ pacman -Syuu
 
 Follow the instructions. Repeat this step until it says there are no packages to update.
 See the above link if you have an older installation of MSYS2 and/or pacman.
-```
-\- Install MSYS2 packages for general build development environment:
-```
-$ pacman -S base-devel git mingw-w64-i686-zlib mingw-w64-i686-toolchain
-```
 
-### Qt-specific steps
+- Install MSYS2 packages for general build development environment:
 
-\- Acquire and install the qt5-static library environment:
-```
-$ pacman -S mingw-w64-i686-qt5-static
 ```
-\- Modify `.bash_profile` to add qt5-static binaries to `PATH`:
-```
-$ echo 'if [[ "${MSYSTEM}" == "MINGW"* ]]; then PATH="/${MSYSTEM}/qt5-static/bin:${PATH}"; fi' >> ~/.bash_profile
+pacman -S base-devel \
+          git \
+          mingw-w64-i686-toolchain \
+          mingw-w64-i686-cmake \
+          mingw-w64-i686-ninja \
+          mingw-w64-i686-zlib
 ```
 
 ### Testrunner-specific steps
 
-\- Install the `libpng` package:
+- Install the `libpng` and `python3` packages:
+
 ```
-$ pacman -S mingw-w64-i686-libpng
+pacman -S mingw-w64-i686-libpng mingw-w64-i686-python3
 ```
 
+### Visual Studio users
+
+As there is no standard package manager for Visual Studio, you will need to either manually build zlib (and libpng for test), or use a package manager such as [vcpkg](https://github.com/microsoft/vcpkg).
+
+You will also need to manually install [CMake](https://cmake.org/) and to make sure to add it to your `PATH` during installation.

```

Comment:
CMake is one of the possible components which can be installed with the Visual Studio installer (C++ CMake tools for Windows) which also includes Ninja.

Reply:
Thank you for the feedback, I've updated the documentation appropriately. It's a similar situation to Msys2 shell where the tools are only available from a specific shell (Developer PowerShell for VS).